doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -1,68 +1,60 @@
-- Connecting to the serial port, described in :ref:`Connecting to the serial port
-  <connect_serial_port>`
-- Install Grub on SATA or NVME disk of NUC
+- Use the `Intel NUC Kit NUC7i7DNHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i7dnhe.html>`_.
+- Connect to the serial port as described in :ref:`Connecting to the serial port <connect_serial_port>`.
+- Install GRUB on your SATA device or on the NVME disk of your NUC.
 
-Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
-***********************************************************
-#. Append the following configuration to the ``/etc/grub.d/40_custom`` file:
+Update Ubuntu GRUB
+******************
+
+Perform the following to update Ubuntu GRUB so it can boot the hypervisor and load the kernel image:
+
+#. Append the following configuration in the ``/etc/grub.d/40_custom`` file:
 
    .. code-block:: bash
-      :emphasize-lines: 18,20
+      :emphasize-lines: 10,11
 
       menuentry 'ACRN hypervisor Hybird Scenario' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
-
          recordfail
-
          load_video
-
          gfxmode $linux_gfx_mode
-
          insmod gzio
-
          insmod part_gpt
-
          insmod ext2
-
          echo 'Loading hypervisor Hybrid scenario ...'
-
          multiboot --quirk-modules-after-kernel /boot/acrn.32.out
-
          module /boot/zephyr.bin XXXXXX
-
          module /boot/bzImage yyyyyy
 
       }
 
-   .. note:: The module ``/boot/zephyr.bin`` is VM0(Zephyr) kernel file,
-      param ``xxxxxx`` is VM0’s kernel file tag and must exactly match the ``kernel_mod_tag``
-      of VM0 which configured in file ``hypervisor/scenarios/hybrid/vm_configurations.c``.
-      The multiboot module ``/boot/bzImage`` is Service VM kernel file, param ``yyyyyy``
-      is the bzImage tag and must exactly match the ``kernel_mod_tag`` of VM1 in file
-      ``hypervisor/scenarios/hybrid/vm_configurations.c``.
-      The kernel command line arguments used to boot the Service VM is located in header file
-      ``hypervisor/scenarios/hybrid/vm_configurations.h`` and configured by `SOS_VM_BOOTARGS` MACRO.
+
+   .. note:: The module ``/boot/zephyr.bin`` is the VM0(Zephyr) kernel file.
+      The param ``xxxxxx`` is VM0’s kernel file tag and must exactly match the
+      ``kernel_mod_tag`` of VM0 which is configured in the ``hypervisor/
+      scenarios/hybrid/vm_configurations.c`` file. The multiboot module ``/boot
+      /bzImage`` is the Service VM kernel file. The param ``yyyyyy`` is the
+      bzImage tag and must exactly match the ``kernel_mod_tag`` of VM1 in the
+      ``hypervisor/scenarios/hybrid/vm_configurations.c`` file.
+      The kernel command line arguments used to boot the Service VM is located
+      in the header file ``hypervisor/scenarios/hybrid/vm_configurations.h`` and is configured by `SOS_VM_BOOTARGS` MACRO.
```

Comment:
similarly, change ``is configured`` to ``are configured``, and I'd keep the line length < 80 here too.